addresses.go

```diff
@@ -77,10 +89,15 @@ func (ipaddress IPAddress) Delete(ctx context.Context, client *Client) error {
 
 // AssociateIPAddress (Async) represents the IP creation
 type AssociateIPAddress struct {
-	IsPortable *bool `json:"isportable,omitempty" doc:"should be set to true if public IP is required to be transferable across zones, if not specified defaults to false"`
-	NetworkdID *UUID `json:"networkid,omitempty" doc:"The network this ip address should be associated to."`
-	ZoneID     *UUID `json:"zoneid,omitempty" doc:"the ID of the availability zone you want to acquire an public IP address from"`
-	_          bool  `name:"associateIpAddress" description:"Acquires and associates a public IP to an account."`
+	Interval    int64  `json:"interval,omitempty" doc:"healthcheck definition: time in seconds to wait for each check. Default: 10, minimum: 5"`
```

Comment:
I'd prefix the healthcheck-related struct fields with `HealthcheckXXX` for a more explicit properties separation (even if the actual API parameters are not).

Reply:
Done ;)

Reply:
Perfect! 👌